core/dbt/adapters/base/relation.py

```diff
@@ -433,13 +433,14 @@ def search(
             for schema in schemas:
                 yield information_schema_name, schema
 
-    def flatten(self):
+    def flatten(self, allow_duplicates: bool=False):
```

Comment:
Just a naming nit: this change isn't so much about duplicates per se; it's about allowing multiple databases at all. In the past, the only reason we'd expect to see multiple databases on Postgres/Redshift was if a metadata query accidentally returned duplicates.